skillpacks/base.py

```diff
@@ -76,6 +77,29 @@ def __init__(
         self.hidden = hidden
         self.episode_id = episode_id
 
+        # Set action_id for each ActionOpt
+        for action_opt in self.action_opts:
+            action_opt.action_id = self.id
+
+    def add_actionOpt(
+        self,
+        action: V1Action,
+        prompt: Optional[Prompt] = None,
+        reviews: List[Review] = [],
+        created: Optional[float] = None,
+        updated: Optional[float] = None,
+    ) -> None:
+        actionOpt = ActionOpt(
```

Comment:
get these javascript variable casings out of here! 😛

Reply:
it looks similar!